easybuild/easyconfigs/q/QBOX/QBOX-r140b-foss-2016a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'QBOX'
+version = 'r140b'
+
+homepage = 'https://asc.llnl.gov/CORAL-benchmarks/#qbox'
+description = 'QBOX benchmark, part of CORAL suite'
+
+toolchain = {'name': 'foss', 'version': '2016a' }
+
+source_urls = ['https://asc.llnl.gov/CORAL-benchmarks/Science/']
+sources = ['qball_%(version)s.tgz']
+
+checksums = ['3eef50f376b02f9d569e0d6e761847ce']
+
+patches = ['QBOX-r140b_fix.patch']
+
+files_to_copy = ['trunk']
```

Comment:
maybe copy the binary to `bin/` instead?

```python
files_to_copy = [(['trunk/src/qb-linux_x86_64_intel'], 'bin')]
```

(and then adjust accordingly below: drop `modextrapaths` since `bin` is added to `$PATH` by default, update `sanity_check_paths`